### ISSUE TITLE
MDN boxes obscure text when printed

### ISSUE THREAD

**Issue by marcoscaceres** [speced/bikeshed#2840]
When a spec is printed, the MDN boxes sometimes cover the text. To reproduce, print a PDF of, for instance:
https://www.w3.org/TR/orientation-event/

Here is an example: 

![Screenshot 2024-04-22 at 1 50 27 PM](https://github.com/speced/bikeshed/assets/870154/f59f47c3-4786-4393-86d8-6a4a707d64c7)

**Comment by marcoscaceres** [comment]
As these elements are non-interactive, I wonder if we should remove them when printing? Not sure.

**Comment by tabatkins** [comment]
Yeah, I think they should just be hidden while printing. *In theory* their information could be useful, but it seems *really* niche, and the display issues with doing so are large enough that I don't think it's remotely worth it.
